org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserProjectFileManager.java

```diff
@@ -27,11 +29,10 @@ public UserProjectFileManager(String baseUrl, String[] fileNames) {
    */
   public void loadFiles() throws UserFacingException, UserInitiatedException {
     HttpClient client = HttpClient.newBuilder().build();
-    // TODO: Enable multi-file. For now, we will always have exactly one file.
-    String fileName = fileNames[0];
+    // TODO: Support loading validation code
     HttpRequest request =
         HttpRequest.newBuilder()
-            .uri(URI.create(String.join("/", baseUrl, fileName)))
+            .uri(URI.create(String.join("/", baseUrl, MAIN_SOURCE_FILE_NAME)))
```

Comment:
This join logic originally existed because we were using the files api and didn't know what the user files would be named or how many files there would be. Now that we're using the sources API, the client should go ahead and pass the URL for the main.json in its full form.

Reply:
we will eventually also need to parse validation.json--do you image we will pass both full urls as parameters?

Reply:
Changing this comment to non-blocking. There's an [offline conversation](https://codedotorg.slack.com/archives/C01EF4GJ9GE/p1618437424028900) regarding validation.json going on in slack which shouldn't block this PR.